docs/walkthrough_transcribe.md

```diff
@@ -2,22 +2,96 @@
 
-!!! warning
-    The section needs to be completed.
 
+Once the [segmentation of the images](walkthrough_segment.md) have been achieved and [some annotations](walkthrough_annotate.md) have been added, if needed, the next step in eScriptorium will be the text recognition of the documents.  
+This step can be done manually or automatically. Automatic text recognition is performed with a model (although you may still need some manual correction afterwards). Manual transcription can be done by typing to the text directly in the interface, by copying and pasting text or by [importing XML transcriptions](walkthrough_import.md).
 ## Manual transcription
+For the manual transcription, two panels can be used: "Transcription", that will display the transcription as it is structured in the images, and "Text", that will display the transcription as a plain succession of text lines, according to the order from the segmentation.
+
+![image: Transcription panel on eScriptorium (access button framed in red)](img/transcribe/transcription_panel.png "Transcription panel on eScriptorium (access button framed in red)")
+
+![image: Text panel on eScriptorium (access button framed in red)](img/transcribe/text_panel.png "Text panel on eScriptorium (access button framed in red)")
+
+### Transcription by hand
+When the transcription has to be done by hand, the best option is to use the "Transcription" panel.  
+To start transcribing, click on the corresponding zone. An input window is displayed. To record a transcription, press “Enter”: the transcription interface automatically displays the input field for the next segment.
+
+![image: Illustration of the transcription of the text by hand.](img/transcribe/by_hand.gif "Illustration of the transcription of the text by hand")
+
+### Transcription by copy/paste
+When the only available format of the transcription is text, it is possible to add it swiftly on eScriptorium.  
+To do so, make sure that the "Text" pane is open. Then, from the text file of your transcription, copy the portion of text present in the image. You can then check, with the "Transcription" pane if everything correspond to the image.
+
+![image: Illustration of the transcription of the text via copy/paste.](img/transcribe/via_copy.gif "Illustration of the transcription of the text via copy/paste")
+
+!!! Note
+    If the transcription does not match the segmentation of the image, you have the possibility to modify the transcription in the lines through one of the two panes, until it matches completly.
 
 ## Automatic transcription
+1. You need a model to transcribe automatically;
+2. There are different ways to have a model: [training on escriptorium](walkthrough_train.md), [share a model](walkthrough_train.md), import a model either (because the model can be shared on [Zenodo](https://zenodo.org/communities/ocr_models/), can be trained in command line with [kraken](https://kraken.re), or can be sent to you by someone else as a file).
```

Comment:
Error in the link for 'share a model' --> need to modify it later to put the right link

Reply:
Ah sorry, I was just breaking down the argument in this section, you don't have to make a numbered list 😅

Reply:
Actually a bullet list would work well when you list the different ways to have access to a model